docs/guide/html-and-static-assets.md

```diff
@@ -39,6 +39,11 @@ By default, a Vue CLI app will automatically generate prefetch hints for all Jav
 
 The hints are injected using [@vue/preload-webpack-plugin](https://github.com/vuejs/preload-webpack-plugin) and can be modified / deleted via `chainWebpack` as `config.plugin('prefetch')`.
 
+::: tip 
+Note for multi page setups
```

Comment:
The header line should be placed right after `::: tip`, instead of starting a new line
See the difference in format:
![image](https://user-images.githubusercontent.com/3277634/54486771-f81f3f80-48c7-11e9-9c31-93a6e45d6554.png)
![image](https://user-images.githubusercontent.com/3277634/54486773-fce3f380-48c7-11e9-8eb7-ce0e1f1b9008.png)

Reply:
Ok no problem, all done

Reply:
Thanks! 😄
I'll merge it soon.